cyclops-ctrl/api/v1alpha1/module_types.go

```diff
@@ -65,7 +65,8 @@ type ReconciliationStatus struct {
 	// +kubebuilder:validation:Optional
 	Reason string `json:"reason,omitempty"`
 	// +kubebuilder:validation:Optional
-	Errors []string `json:"errors"`
+	Errors     []string `json:"errors"`
+	FinishedAt string   `json:"finishedAt,omitempty"`
```

Comment:
Can you add `// +kubebuilder:validation:Optional` comment above finishedAt. Same goes for finishedAt in HistoryEntry